### PR TITLE
Update Korean translation

### PR DESCRIPTION
**Pull request by marocchino**
@yous review please. :)

### CODE REVIEW

File: languages/ko.json [hunk 1]
```diff
@@ -113,6 +115,10 @@
   "workshopper-learnyoubash": "터미널을 사용하는 방법과 첫 Bash 스크립트를 작성하는 법을 배우세요.",
   "workshopper-currying": "커링 컨셉과 JavaScript에서 커링 함수를 만드는 법을 배우세요.",
   "workshopper-seneca-in-practice": "SenecaJs 마이크로서비스 툴킷을 활용하는 법을 배우세요.",
+  "workshopper-bacon-love": "Bacon.js 라이브러리를 통해 함수형과 리엑티브 프로그래밍의 개념을 배우세요.",
```
Comment (yous) [inline languages/ko.json]: 리엑티브 → 리액티브

File: languages/ko.json [hunk 2]
```diff
@@ -113,6 +115,10 @@
...
+  "workshopper-innersourceadventure": "InnerSource 소프트웨어 개발을 배우세요.",
+  "workshopper-less-is-more": "LESS CSS 전처리기의 기초를 배우세요.",
+  "workshopper-js-best-practices": "깔끔한 JavaScript 코드를 작성하기 위한 베스트 프랙티스를 배우세요.",
```
Comment (yous) [inline languages/ko.json]: How about '깔끔한 JavaScript 코드 작성의 모범 사례를 배우세요.', or just '모범 사례' instead of '베스트 프랙티스'?

**Comment by yous** [comment]
Can you review `index-header-bottom` and `index-header-bottom{A,B,C,D}` also?

```json
"index-header-bottom": "",
"index-header-bottomA": "핵심 워크숍",
"index-header-bottomB": "설치를 통해 바로 시작하거나",
"index-header-bottomC": "이메일 뉴스레터",
"index-header-bottomD": "를 무료로 구독하세요",
```

**Comment by marocchino** [comment]
@yous did it. >_<

File: languages/ko.json [hunk 3]
```diff
@@ -148,7 +155,7 @@
   "about-header-touch-title": "함께 작업하기",
   "about-info2": "우리는 NodeShcool 구성을 위해 GitHub를 사용하고 있습니다. 공동작업을 위한 최고의 공간은 GitHub의 저장소입니다. 이곳에서 새로운 이슈를 생성할 수 있고 그 이슈는 모두에게 알람이 가고 응답을 받으실 수 있습니다. 이 방법은 구성원들에게 바로 이메일을 보내는 것보다 훨씬 좋은 방법입니다.",
   "about-header-talk-title": "사람들의 이야기를 듣고 보기",
-  "about-info3": "NodeUp podcast Mikeal Rogers의 에피소드 55 에서는, Max Ogden 과 다른 커뮤니티 멤버들이 NodeSchool에 대해 이야기를 하고 있습니다. 볼티모어에서 개최된 At Cascadia JS 2014 에서는 Jason Rhodes 가 NodeSchool에 대해 얘기 했습니다",
+  "about-info3": "NodeUp podcast Mikeal Rogers의 에피소드 55 에서는, Max Ogden 과 다른 커뮤니티 멤버들이 NodeSchool에 대해 이야기를 하고 있습니다. 볼티모어에서 개최된 At Cascadia JS 2014 에서는 Jason Rhodes 가 NodeSchool에 대해 얘기 했습니다.",
```
Comment (yous) [inline languages/ko.json]: > `In <a href="https://archive.org/details/NodeUp55" target="_blank" rel="noreferrer noopener">episode 55</a> of the NodeUp podcast Mikeal Rogers, Max Ogden and other community members talk about NodeSchools. At Cascadia JS 2014 Jason Rhodes, from Baltimore, <a href="https://www.youtube.com/watch?v=YJ7txKTh3-E" target="_blank" rel="noreferrer noopener">talks about running NodeSchools</a>.`

`NodeUp 팟캐스트의 <a href="https://archive.org/details/NodeUp55" target="_blank" rel="noreferrer noopener">에피소드 55</a>에서는 Mikeal Rogers, Max Ogden, 다른 커뮤니티 멤버들이 NodeSchool에 대해 이야기를 하고 있습니다. Cascadia JS 2014에서는 볼티모어의 Jason Rhodes가 <a href="https://www.youtube.com/watch?v=YJ7txKTh3-E" target="_blank" rel="noreferrer noopener">NodeSchool 운영에 대해 얘기했습니다</a>.`

**Comment by jpwesselink** [comment]
🎉

**Comment by marocchino** [comment]
@yous did it again. >_<

**Pull request merged**